packages/components/src/internal/components/entities/models.ts

```diff
@@ -538,34 +548,52 @@ export interface IEntityTypeDetails extends IEntityDetails {
 }
 
 export interface EntityDataType {
-    typeListingSchemaQuery: SchemaQuery; // The schema query used to get the listing of all of the data type instances (e.g., all the data classes) available
-    listingSchemaQuery: SchemaQuery; // The schema query used to get the listing of all of the data instances (e.g., all the data class rows) available
-    instanceSchemaName: string; // (e.g., samples) Name of the schema associated with an individual instance that can be used in conjunction with a name returned from the typeListingSchemaQuery listing
-    operationConfirmationActionName: string; // action in ExperimentController used to get the confirmation data for performing operations on entities
-    nounSingular: string;
-    nounAsParentSingular: string;
+    // used for extracting or querying for the parents of this type
+    ancestorColumnName: string;
+    appUrlPrefixParts?: string[];
+    // A list of fields that are backed by ExprColumn and the ExprColumn's sql contain sub select clauses
```

Comment:
eslint does not handling moving comments well and it's a known issue:
https://github.com/import-js/eslint-plugin-import/issues/1723
https://github.com/import-js/eslint-plugin-import/issues/1450

Instead of adding `eslint-disable-line` to all lines, I went ahead with moving all inline comments to a block comment above the interface. 
Let me know if anyone has better ideas. @labkey-nicka @cnathe @labkey-alan